src/averageAges.js

```diff
@@ -16,6 +16,18 @@
  */
 function calculateMenAverageAge(people, century) {
   // write code here
+  let ages;
+  const mens = people.filter(person => person.sex === 'm');
+
+  if (century !== undefined) {
+    ages = [...mens]
+      .filter(men => Math.ceil(men.died / 100) === century)
+      .map(men => men.died - men.born);
+  } else {
+    ages = [...mens].map(men => men.died - men.born);
+  }
```

Comment:
`Array.filter()` returns new array, no need creting a copy.
Use ternary operator instead of `if else` statement